src/components/Types.ts

```diff
@@ -2,6 +2,8 @@ export type TypeState = {
   drinkList: Drink[];
   id?: string;
   error?: false | true;
+  setDrinkList?: (active: Drink[]) => void;
```

Comment:
ahh, I see how this can be truck to navigate with Hooks and TS. We can talk about more ideas tomorrow maybe? -Jordan